app/src/main/kotlin/compiler/lowlevel/linearization/Linearization.kt

```diff
@@ -77,34 +76,14 @@ object Linearization {
                     dfs(targetWhenTrue, label)
                     dfs(targetWhenFalse, nextLabel)
                 }
-            } else if (node in cfg.conditionalTrueLinks) {
```

Comment:
I mentioned about this here: https://github.com/TCS-Compilers/compiler/pull/114#discussion_r1045985093, but I guess we can remove this for now.